vowpalwabbit/core/src/reductions/epsilon_decay.cc

```diff
@@ -246,7 +246,9 @@ size_t read_model_field(io_buf& io, VW::reductions::epsilon_decay::epsilon_decay
 {
   size_t bytes = 0;
   epsilon_decay.conf_seq_estimators.clear();
+  epsilon_decay._weight_indices.clear();
   bytes += read_model_field(io, epsilon_decay.conf_seq_estimators);
+  bytes += read_model_field(io, epsilon_decay._weight_indices);
```

Comment:
shouldn't a runtest fail?

Reply:
I guess its mostly unit tests

Reply:
yea its just unit tests, doesnt change any of the outcomes